### PR TITLE
Pattern Library: Design tweaks

### PR DESCRIPTION
**Pull request by fredrikekelund**
Fixes https://github.com/Automattic/dotcom-forge/issues/6491

## Proposed Changes

1. Use `preventWidows` instead of `text-wrap: pretty` because of browser support
2. Use a `min-height` for the `Copy, paste, customize` card headings, to improve layout alignment
3. Use a white background color and black text color for `Get started` CTA in main header (per Figma mockups)
4. Tweak height and background color of pattern type and view toggles (per Figma mockups)
5. `text-transform: capitalize` for copy buttons (for consistency with other buttons)

| Before | After |
| - | - |
| before | after |
| before | after |

## Testing Instructions

1. Make sure everything looks good on desktop and on mobile

### CODE REVIEW

**Comment by matt-west** [comment]
Looks good. Thanks @fredrikekelund!

File: client/my-sites/patterns/components/pattern-library/index.tsx
Comment (stephanethomas) [inline client/my-sites/patterns/components/pattern-library/index.tsx:349]: Not related to your changes but that's the first time I see double dashes `--` in class names. I was curious, and here is what I found about that BEM feature:

> Modifiers indicate a different version or state of a block or an element. They are used to style the block or element differently from the default. Modifiers are denoted by double hyphens -- following the name of the block or element.

I wouldn't necessary introduce a new class for that, but that may just be personal preference. Feel free to leave it as it-is.

**Review by stephanethomas — APPROVED** [review]
> Use `preventWidows` instead of `text-wrap: pretty` because of browser support

Should we also apply that change to other pieces of text? I've found an issue with French:

image

> Use a white background color and black text color for `Get started` CTA in main header (per Figma mockups)

That may just be me but I would retain the blue color - this page is already very black and white, and this is an important call-to-action which is also in blue on other landing pages. I'll let you decide though.

> `text-transform: capitalize` for copy buttons (for consistency with other buttons)

It looks like we also need to capitalize those buttons:

image

Apart from that, this is looking good and doesn't require another review.

**Comment by fredrikekelund** [comment]
> Should we also apply that change to other pieces of text?

Good call with the body text of those cards 👍

> I would retain the blue color

The white color is what we have in the design. If we don't want the white, then maybe we should go for the same blue that we have in the category nav and in the copy buttons. @matt-west, WDYT?

> capitalize those buttons

I wanted to capitalize `Copy pattern` because it's rendered so close to the category nav, where the category names are capitalized. 

In this case, I _think_ the preferred capitalization is `Create a Free Account`, which means that `text-transform: capitalize` doesn't work since it would yield `Create A Free Account`. Since the other text elements in the modal are also not capitalized, I think it's OK to stick with the current approach here.

Btw, I updated the capitalization rule for `Copy Pattern` to only apply in English.

**Comment by stephanethomas** [comment]
> Btw, I updated the capitalization rule for `Copy Pattern` to only apply in English.

Introducing a rule that is specific to the English language doesn't sound right to me. This create a discrepancy which may confuse us down the line.

> I wanted to capitalize `Copy pattern` because it's rendered so close to the category nav, where the category names are capitalized.

Category names are not capitalized in French and in some other languages. We discuss this issue recently with Matt in  p1712242735704369-slack-C06ELHR6L9J. We could use CSS to fix that though, even if it is imperfect, as you noted. I personally don't see a problem with capitalizing each first letter (aka `Create A Free Account`).

**Comment by matt-west** [comment]
> I would retain the blue color

I went with the white button here to be consistent with the header on the home page, and what we have planned for the new /plugins page.

Screenshot 2024-04-09 at 15 53 37

I think we should address the concern about the page being too black and white separately. Adding the ability to apply a colorset to pattern previews would definitely help with that. I'd also like to explore adding more educational content to the page which would be a good opportunity to introduce more color.

**Comment by fredrikekelund** [comment]
> Introducing a rule that is specific to the English language doesn't sound right to me

I think capitalization rules and conventions are highly dependent on the language. For example, it's uncommon in Swedish to capitalize stylistically, as you often see in English. In German, all nouns are capitalized, making it (presumably) weird to see all words capitalized.

I'll let you speak to the French case, @stephanethomas, but I would have guessed that stylistic capitalization is less common for you, too. If nothing else, neither the `Patterns / Page layouts` toggle, nor the multi-word category names in the pattern library are capitalized in French, and I'd say consistency is the main goal here.

**Comment by fredrikekelund** [comment]
I'm happy to continue this discussion, @stephanethomas, but I'll go ahead and merge this PR so we get it out before the launch today

**Pull request merged**
